--- shapelib-1.2.10/contrib/shpgeo.c~	2003-04-07 15:03:22.000000000 +0200
+++ shapelib-1.2.10/contrib/shpgeo.c	2005-06-16 15:48:21.000000000 +0200
@@ -543,7 +543,7 @@ int SHPWriteOGisPolygon ( WKBStreamObj *
    nextring = 0;
    cParts=0;
    while ( nextring >= 0 ) {
-     (SHPObject*) ppsC[cParts] = SHPUnCompound ( psCShape, &nextring ); 
+     SHPUnCompound ( psCShape, &nextring ); 
      cParts++;
     }
    
